specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-02-01/managedClusters.json

```diff
@@ -7322,6 +7326,27 @@
         }
       }
     },
+    "ManagedClusterMetricsProfile": {
+      "type": "object",
+      "description": "The metrics profile for the ManagedCluster.",
+      "properties": {
+        "costAnalysis": {
+          "$ref": "#/definitions/ManagedClusterCostAnalysis",
+          "title": "The configuration for detailed per-Kubernetes resource cost analysis."
+        }
+      }
+    },
+    "ManagedClusterCostAnalysis": {
+      "type": "object",
+      "description": "The cost analysis configuration for the cluster",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "title": "Whether to enable cost analysis",
+          "description": "The Managed Cluster sku.tier must be set to 'Standard' to enable this feature. Enabling this will add Kubernetes Namespace and Deployment details to the Cost Analysis views in the Azure portal. If not specified, the default is false. For more information see aka.ms/aks/docs/cost-analysis."
```

Comment:
Should this include 'Premium' sku.tier as well now?

Reply:
No that other PR is for -preview. I fixed this locally but forgot to push. Fixed - good catch.

Reply:
Basically what I had meant to do was fix it here inline for GA, and then fix it also in the `-preview` API (that other PR), so that GA and preview were consistent.